Package.swift

```diff
@@ -129,6 +129,10 @@ let package = Package(
       name: "FirebaseRemoteConfigSwift",
       targets: ["FirebaseRemoteConfigSwift"]
     ),
+    .library(
+      name: "FirebaseSessions",
+      targets: ["FirebaseSessions"]
+    ),
```

Comment:
Will this library go out for release? If we'd like to keep the SDK unreleased, can we leave off the library but keep the target?

Maybe it's ok to leave the library because customers will have to depend on it directly?

Reply:
This SDK is for internal use only, right? If so, it shouldn't have a library– only a target.

Reply:
AQS Library is internal - yes. Similar to Firelog, other libraries would depend on this similar to Crashlytics/Fireperf. Probably, we might have to do something like this. https://github.com/firebase/firebase-ios-sdk/blob/master/Package.swift#L155-L158

Reply:
Also, we might have have to bring in dependencies for this package? Like Installations and GoogleUtilities?

Reply:
These dependencies are specified in the target, so we're already gtg there: https://github.com/firebase/firebase-ios-sdk/pull/10572/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR1093-R1097

To your comment above, we are ok leaving that out because `.package(` specifications are for libraries outside of the repo. Since we've put the `.target(` in for FirebaseSessions other SDKs can just depend on that target.

Reply:
Going to remove this library specification